core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -350,6 +350,17 @@
   MASTER_METADATA_SUSPENDABLE("master.metadata.suspendable", "false", PropertyType.BOOLEAN,
       "Allow tablets for the " + MetadataTable.NAME
           + " table to be suspended via table.suspend.duration."),
+  MASTER_STARTUP_TSERVER_AVAIL_MIN_COUNT("master.startup.tserver.avail.min.count", "0",
+      PropertyType.COUNT,
+      "Minimum number of tservers that need to be registered before master will "
+          + "start tablet assignment - checked when master gets lock. To disable check, set to <= 0. "
```

Comment:
Could provide a little more info than `checked when master gets lock`.  Let users know that this prop is only checked once at startup after the lock is acquired.